src/components/config-pages/global-message.tsx

```diff
@@ -24,38 +22,30 @@ const GlobalMessage = ({
   });
 
   return (
-    <div className={wrapperClasses + " py-10"}>
-      <div className="centered flex flex-col lg:flex-row gap-10">
-        <div className="flex items-center leading-none shrink-0">
-          <MessageIcon messageType={suGlobalMsgType}/>
-          {suGlobalMsgLabel}:
-        </div>
-        <div className="[&_a]:text-white [&_a.btn]:bg-transparent [&_a.btn]:border-2 [&_a.btn]:border-white">
-          {suGlobalMsgHeader && <H2>{suGlobalMsgHeader}</H2>}
-
-          <Wysiwyg html={suGlobalMsgMessage?.processed}/>
+    <div className='bg-fog-light'>
+      <div className='md:centered'>
```

Comment:
Double quotes. use `yarn lint` to catch these.

Reply:
For some reason when I run `yarn lint`, it doesn't throw an error for single quotes 🤔 

```
$ yarn lint
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.3.5 <5.4.0

YOUR TYPESCRIPT VERSION: 5.4.5

Please only submit bug reports when using the officially supported version.

=============
✔ No ESLint warnings or errors
.storybook/stories/elements/lockup/Lockup.stories.tsx:35:17 - error TS2739: Type '{ url: string; height: number; width: number; size: number; }' is missing the following properties from type 'Image': focalX, focalY

35     if(logoUrl) args.suUploadLogoImage = {url: logoUrl, height: 50, width: 300, size: 123}
                   ~~~~~~~~~~~~~~~~~~~~~~

.storybook/stories/storybook-entities.tsx:24:5 - error TS2739: Type '{ url: string; height: number; width: number; alt: string; size: number; }' is missing the following properties from type 'Image': focalX, focalY

24     mediaImage: {
       ~~~~~~~~~~

  src/lib/gql/__generated__/drupal.d.ts:613:3
    613   mediaImage: Image;
          ~~~~~~~~~~
    The expected type comes from property 'mediaImage' which is declared here on type 'MediaImage'

.storybook/stories/storybook-entities.tsx:56:5 - error TS2739: Type '{ url: string; height: number; width: number; alt: string; size: number; }' is missing the following properties from type 'Image': focalX, focalY

56     suGalleryImage: {
       ~~~~~~~~~~~~~~

  src/lib/gql/__generated__/drupal.d.ts:666:3
    666   suGalleryImage?: Maybe<Image>;
          ~~~~~~~~~~~~~~
    The expected type comes from property 'suGalleryImage' which is declared here on type 'MediaStanfordGalleryImage'


Found 3 errors in 2 files.

Errors  Files
     1  .storybook/stories/elements/lockup/Lockup.stories.tsx:35
     2  .storybook/stories/storybook-entities.tsx:24
```

Reply:
Merge/rebase the latest of `1.x` and you should be good.